### PR TITLE
Features/travis

### LINKED ISSUE #6: Add CI using Travis
We should add CI to this role using the stack below:

- Ansible
- Vagrant
- Molecule

We should configure Travis to run the unit tests in each push/merge: http://molecule.readthedocs.io/en/stable-1.18/usage.html

### PR DESCRIPTION
**Pull request by jnogol**
Fixes #6 
Corrects tiny typo

### CODE REVIEW

**Comment by jnogol** [comment]
Build in Travis is failing though in a local environment works perfectly (I know, sounds like an excuse xD). I took a look on the playbook and found that this task https://github.com/idealista/nginx-role/blob/develop/tasks/install.yml#L76 can be split in three separate ones. Shall we do this? @dortegau @jmonterrubio

**Comment by dortegau** [comment]
I would prefer this task splitted in three 👍

File: tasks/install.yml [hunk 1]
```diff
@@ -73,12 +78,32 @@
   with_items:
     - "{{ nginx_compile_time_options_external_modules_paths }}"
 
-- name: NGINX | Install nginx
-  command: "chdir=/tmp/nginx-{{ nginx_version }} {{item}}"
-  with_items:
-    - "./configure {{ nginx_compile_time_options_str }} {{ nginx_compile_time_options_builtin_modules_str }} {{ nginx_compile_time_options_external_modules_paths_str }}"
-    - make
-    - make install
+# - name: NGINX | Install nginx
```
Comment (jmonterrubio) [inline tasks/install.yml]: I'ld remove the commented lines

File: defaults/main.yml [hunk 2]
```diff
@@ -58,5 +58,5 @@ nginx_compile_time_options_builtin_modules:
   - http_stub_status_module
 
 nginx_compile_time_options_external_modules_paths:
-  - /tmp/headers-more-nginx-module-{{ headers_more_version }}
-  - /tmp/lua-nginx-module-{{ lua_module_version }}
+  - /nginxbuild/headers-more-nginx-module-{{ headers_more_version }}
```
Comment (dortegau) [inline defaults/main.yml]: [FSH (Filesystem Hierarchy Standard)](http://www.pathname.com/fhs/pub/fhs-2.3.html#PURPOSE18) suggests to put those source files and folders under _/usr/src_. Anyway, I suggest you to use a variable to store this path.

File: molecule.yml [hunk 3]
```diff
@@ -60,3 +60,36 @@ vagrant:
         - network_name: private_network
           type: dhcp
           auto_config: true
+
+docker:
+  containers:
+    - name: nginx.vm
```
Comment (dortegau) [inline molecule.yml]: That _.vm_ TLD is not needed in docker :)

**Comment by dortegau** [comment]
There are conflicts in molecule.yml file 😄

File: tests/roles/nginx-old/tasks/main.yml [hunk 4]
```diff
@@ -10,42 +10,67 @@
...
-        --prefix=/usr
-        --add-module=/tmp/headers-more-nginx-module-{{ headers_more_version }}
-        --conf-path={{ nginx_conf }}
-        --error-log-path={{ nginx_err }}
-        --http-log-path={{ nginx_acc }}
-        --pid-path=/var/run/nginx.pid
-        --lock-path=/var/lock/nginx.lock
-    - make
-    - make install
+# - name: Install nginx
```
Comment (dortegau) [inline tests/roles/nginx-old/tasks/main.yml]: I'd remove this commented lines too :D

**Comment by jnogol** [comment]
Tests are broken (because Testinfra), but are fixed in `features/goss`

**Pull request merged**